src/main/java/uk/gov/hmcts/divorce/divorcecase/model/WhoPaysCostOrder.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.divorce.divorcecase.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum WhoPaysCostOrder implements HasLabel {
```

Comment:
We should ask about this because RESPONDENT_AND_CO_RESPONDENT and CO_RESPONDENT are no longer relevant